app/assets/javascripts/snowcrash/modules/issues/table.js.coffee

```diff
@@ -1,218 +1,31 @@
-        issue_ids.push($(this).val())
-
-      $.ajax $('#issue-table').data('destroy-url'), {
-        method: 'POST'
-        dataType: 'json'
-        data: {ids: issue_ids}
-        success: ->
-          for id in issue_ids
-            $("input#issue_#{id}").closest('tr').remove()
-            $("#issue_#{id}").remove()
-
-          if $(@selectedIssuesSelector).length == 0
-            that.resetToolbar()
-
-        error: ->
-          for id in issue_ids
-            $row = $("input#issue_#{id}").closest('tr')
-            $($row.find('td')[2]).replaceWith("<td class='text-error'>Please try again</td>")
-      }
-
-    # prevent Rails UJS from doing anything else.
-    false
-
-  onTagSelected: (event) =>
-    that = this
-    $target = $(event.target)
-    event.preventDefault()
-
-    $('.js-tbl-issues').find(@selectedIssuesSelector).each ->
-      $this = $(this)
-
-      $row = $this.parent().parent()
-      $($row.find('td')[that.tagColumnIndex]).replaceWith("<td class=\"loading\">Loading...</td>")
-
-      url   = $this.data('url')
-      data  = { issue: { tag_list: $target.data('tag') } }
-      $that = $this
-
-      $.ajax url, {
-        method: 'PUT',
-        data: data,
-        dataType: 'json',
-        success: (data) ->
-          $that.prop('checked', false)
-          issue_id = $that.val()
-
-          $($row.find('td')[that.tagColumnIndex]).replaceWith(data.tag_cell)
-          $("#issues #issue_#{issue_id}").replaceWith(data.issue_link)
-          if $(@selectedIssuesSelector).length == 0
-            that.resetToolbar()
-
-        error: (foo,bar,foobar) ->
-          $($row.find('td')[that.tagColumnIndex]).replaceWith("<td class='text-error'>Please try again</td>")
-      }
-
-  onMergeSelected: (event) =>
-    url = $(event.target).data('url')
-    issues_to_merge = []
-
-    $('.js-tbl-issues').find(@selectedIssuesSelector).each ->
-      $row = $(this).parent().parent()
-
-      id = @.name.split('_')[1]
-      issues_to_merge.push(id)
-
-    location.href = "#{url}?ids=#{issues_to_merge}"
-
-  saveColumnState: ->
-    if Storage?
-      localStorage.setItem(@storageKey(), JSON.stringify(@selectedColumns))
-    else
-      console.log "The browser doesn't support local storage of settings."
-      console.log "Column selection can't be saved."
-
-  showHideColumns: =>
-    that = this
-
-    $.each @$table.find('thead th'), (index, th)->
-      $th = $(th)
-
-      if (column = $(th).data('column'))
-        that.tagColumnIndex ||= index if column == 'tags'
-        if that.selectedColumns.indexOf(column) > -1
-          that.$table.find("td:nth-child(#{index + 1})").css('display', 'table-cell')
-          $th.css('display', 'table-cell')
-        else
-          that.$table.find("td:nth-child(#{index + 1})").css('display', 'none')
-          $th.css('display', 'none')
-
-  storageKey: ->
-    @project ||= $('.brand').data('project')
-    "project.#{@project}.issue_columns"
-
 jQuery ->
```

Comment:
use `document.addEventListener "turbolinks:load", ->` instead of `jQuery ->`

Reply:
✅